### PR TITLE
Added Version property to project config

### PR DESCRIPTION
**Pull request by ViktorijaAdo**

### CODE REVIEW

File: SecurityCodeScan.Test/Tests/Config/ConfigurationTest.cs [hunk 1]
```diff
@@ -14,8 +14,11 @@ namespace SecurityCodeScan.Test.Config
     public abstract class ConfigurationTest : DiagnosticVerifier
     {
         private readonly List<string> FilePaths = new List<string>();
-        private readonly string ConfigName = "SecurityCodeScan.config.yml";
-        protected async Task<AnalyzerOptions> CreateAnalyzersOptionsWithConfig(string configSource)
+        private const string ConfigName = "SecurityCodeScan.config.yml";
+        private readonly Version ConfigVersion = new Version(1,0);
+        private const string ConfigText = "Version: {0}\n\r{1}";
```
Comment (JarLob) [inline SecurityCodeScan.Test/Tests/Config/ConfigurationTest.cs]: It is \r\n on Windows

File: SecurityCodeScan/Config/ConfigurationManager.cs [hunk 2]
```diff
@@ -278,6 +283,11 @@ public IEnumerable<string> GetAntiCsrfAttributes(ImmutableArray<AdditionalText>
         }
     }
 
+    internal class ProjectConfigData : ConfigData
+    {
+        public string Version { get; set; }
```
Comment (JarLob) [inline SecurityCodeScan/Config/ConfigurationManager.cs]: Could it deserialize to `Version`? I would rather compare versions, than strings

Comment (ViktorijaAdo) [reply SecurityCodeScan/Config/ConfigurationManager.cs]: The simpler would be convert it to version just before using. 
Something like this: 
``` 
if (new Version(projectConfig.Version) != ConfigVersion)
       return null;
```

Comment (JarLob) [reply SecurityCodeScan/Config/ConfigurationManager.cs]: Works for me, please do.

**Pull request merged**